app/Interactive.hs

```diff
@@ -20,52 +20,51 @@ import qualified Juvix.Nets.Bohm              as Bohm
 
 interactive ∷ Context → Config → IO ()
```

Comment:
The arrows weren't changed as far as I can tell, not sure why it's showing these line changes here. I've removed the showVal function to be a show instance so I changed the code here accordingly.

Reply:
It looks like they were changed from Unicode to non-Unicode. It's fine.

Reply:
It could be the autoformater on save being weird, as the spacing for types and constructors has also changed quite drastically